unity/Holo/Assets/Scripts/Logging.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.IO;
+using UnityEngine;
+
+public class Logging : MonoBehaviour
+{
+    string LogFilePath;
+
+    private void Start()
+    {
+        string LogFileName = DateTime.Now.ToString("ddMMyyyy-HHmmss") + "_log.txt";
+
+        string LogFileDir = Path.Combine(Application.persistentDataPath, "Logs");
+        if (!Directory.Exists(LogFileDir))
+        {
+            Directory.CreateDirectory(LogFileDir);
+        }

```

Comment:
No need to check Directory.Exists before CreateDirectory. CreateDirectory is like ForceDirectories in Delphi :), https://docs.microsoft.com/en-us/dotnet/api/system.io.directory.createdirectory?redirectedfrom=MSDN&view=netframework-4.8#System_IO_Directory_CreateDirectory_System_String_ , it creates directory (even parents) if it doesn't exist.